components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -4063,6 +4063,9 @@ protected void checkAccessControlPermission(APIIdentifier identifier) throws API
             String storeVisibilityRoles = apiResource.getProperty(APIConstants.STORE_VIEW_ROLES);
             if (storeVisibilityRoles != null && !storeVisibilityRoles.trim().isEmpty()) {
                 String[] storeVisibilityRoleList = storeVisibilityRoles.split(",");
+                for(int i = 0; i < storeVisibilityRoleList.length; i++) {
```

Comment:
Formatting issues

Reply:
Lets also use enhanced for loop(https://stackoverflow.com/a/18738765/3296350), its a lot cleaner